docker/base/build.sh

```diff
@@ -76,17 +76,6 @@ for TARGET in $TARGETS; do
     # Archive will be for both amd64,x86 and arm64, arm 7
     $GOMOBILE bind --target=android/arm64,android/arm,android/amd64,android/386 -androidapi 21 $X $V "${T[@]}" --ldflags="$V $LD" -o "/build/$NAME.aar" ./$PACK_RELPATH
   fi
-
-  # Check and build for iOS targets
-  if [ $XGOOS == "." ] || [[ $XGOOS == ios* ]]; then
```

Comment:
I imagined when discussing removal of iOS support as: throwing away xcode from the docker image itself to make it more light.

Reply:
No, this is not the only iOS related code.
There's also update_ios.sh file. It dowloads SDK and applies patches to system libraries.